apps/site/styles/locals.css

```diff
@@ -0,0 +1,3 @@
+html[lang='ko'] {
+  @apply break-words break-keep leading-[1.8];
```

Comment:
@AugustinMauroy 
I've made the changes. Since I'm not very familiar with Tailwind, I'd really appreciate it if you could take a look and let me know if there's anything that needs improvement.😀

Reply:
It didn't have to be 1.8. I just chose a value that seemed appropriate from the recommended ones, but I've changed it to `leading-7` as you suggested. I'm still new to using Tailwind, so I wasn't very experienced. Thanks for letting me know!